<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabela_mensal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cliente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Serviço</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Relatório Mensal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_fechar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_excluir_agendamento">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir Selecionado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_faturamento_mensal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Faturamento Mensal: R$: 0,00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
